### PR TITLE
CORTX-32769: Resolved pylint issues for src and scripts folder

### PR DESCRIPTION
**Pull request by SCP-Seagate**
## Problem Statement

-   Problem statement

## Design

-   For Bug, Describe the fix here.
-   For new test addition post the link for design

## Coding

   Checklist for Author

-   [ ]  Coding conventions are followed and code is formatted (e.g. using pycharm in-built formatter)

## Testing

  Checklist for Author

-   [ ]  New/Affected tests are executed
-   [ ]  Attach test execution logs

## Review 

  Checklist for Author

-   [ ]  JIRA number/GitHub Issue added to PR
-   [ ]  PR is self reviewed
-   [ ]  Jira and state/status is updated and JIRA is updated with PR link
-   [ ]  Check if the description is clear and explained

## Documentation

  Checklist for Author

-   [ ]  Changes done to ReadMe

### CODE REVIEW

File: src/libs/tools/warp.py [hunk 1]
```diff
@@ -35,7 +35,7 @@
 class Warp:
     """Warp class for executing given Warp workload."""
 
-    # pylint: disable=too-many-arguments,too-many-locals
+    # pylint: disable=too-many-arguments,too-many-locals,too-many-instance-attributes
```
Comment (DhobaleVishal) [inline src/libs/tools/warp.py:38]: Instead of disabling pylint reported issues, can't we fix them if possible?

**Review by DhobaleVishal — CHANGES_REQUESTED** [review]

**Comment by DhobaleVishal** [comment]
@SCP-Seagate Please fix `DCO Check` and `Codacy Static Code Analysis` for this PR.

**Review by DhobaleVishal — APPROVED** [review]

File: src/libs/tools/warp.py (hunk 1 above)
Comment (DhobaleVishal) [reply src/libs/tools/warp.py:38]: This is big change so, will be taken care later.

**Review by VikasKumarC — APPROVED** [review]
Approve to merge in dev

**Pull request merged**